rust-common.el

```diff
@@ -16,5 +16,11 @@
   :type 'function
   :group 'rust-mode)
 
+(defcustom rust-indent-offset 4
```

Comment:
Instead of moving it here, do you think a defvaralias should be instead created. Something similar to this: https://github.com/psibi/rustic/blob/4dac7f34c30d91e3ecb2602fff7e81b6efdc75d7/rustic.el#L52